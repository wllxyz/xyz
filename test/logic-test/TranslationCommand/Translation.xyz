<i>-->""==><o>-->
    ($TRANSLATION, 
        ($RULE, ($VARIABLE, "<variable>"), ($VARIABLE, "<variable>")($CONSTANT, "CONSTANT")($REMARK, "$REMARK") ), 
        ($RULE, ($VARIABLE, "<variable>"), ($VARIABLE, "<variable>")($CONSTANT, "CONSTANT")($REMARK, "$REMARK") )
    );
