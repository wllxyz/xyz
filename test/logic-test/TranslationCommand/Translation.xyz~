<i>-->""==><o>-->
($EQ, 
    ($IGNORE, 
        (
            (<variable>, <variable>"CONSTANT"$REMARK),
            (<variable>, <variable>"CONSTANT"$REMARK)
        )
    ), 
    ($TRANSLATION, 
        ($RULE, ($VARIABLE, "<variable>"), ($VARIABLE, "<variable>")($CONSTANT, "CONSTANT")($REMARK, "$REMARK") ), 
        ($RULE, ($VARIABLE, "<variable>"), ($VARIABLE, "<variable>")($CONSTANT, "CONSTANT")($REMARK, "$REMARK") )
    )
) 
"\n";
