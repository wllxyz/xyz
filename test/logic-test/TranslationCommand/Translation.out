{destination_rule=>{expression=>[<variable>,C,O,N,S,T,A,N,T,$REMARK],root=><variable>},source_rule=>{expression=>[<variable>,C,O,N,S,T,A,N,T,$REMARK],root=><variable>}}
